src/MOI_wrapper.jl

```diff
@@ -260,20 +260,20 @@ end
 # coef: List of corresponding coefficients
 # d: dimension of set
 # rev: if true, we unscale instead (e.g. divide by √2 instead of multiply for PSD cone)
-function _scalecoef(rows, coef, d, rev)
+function _scalecoef(rows::Union{UnitRange{Int64}, Vector{Int64}}, coef::Vector{Float64}, d::Int64, rev::Bool)
```

Comment:
`Int64` seems not to work on Windows 32 bits (see AppVeyor)

Reply:
I suggest `rows::AbstractVector{<:Integer}, coeff::AbstractVector{<:AbstractFloat}, d::Integer, rev::Bool` signature if we really want to specify those types; `output = copy(coeff)` should be changed to `output = deepcopy(coeff)` to avoid mutation of `coeffs`